synth.metadata

```diff
@@ -3,23 +3,31 @@
     {
```

Comment:
just fyi not sure if you want to merge this or let the generator do it

Reply:
I think the values in here are going to be ok.